src/valkey-check-aof.c

```diff
@@ -556,6 +556,12 @@ int redis_check_aof_main(int argc, char **argv) {
         goto invalid_args;
     }
 
+    /* Check if filepath is longer than PATH_MAX */
+    if (strlen(filepath) > PATH_MAX) {
```

Comment:
shouldn't we bound the search length?

```suggestion
    if (strnlen(filepath, PATH_MAX+1) > PATH_MAX) {
```

Reply:
Filepath will always be null terminated, but it's ok